tests/filesystem/FileTest.php

```diff
@@ -12,7 +12,7 @@ class FileTest extends SapphireTest {
 	public function testLinkShortcodeHandler() {
 		$testFile = $this->objFromFixture('File', 'asdf');
 
-		$parser = new ShortcodeParser();
+		$parser = ShortcodeParser::create();
```

Comment:
I think we should keep it as it is here, as in the tests we want to be fully sure of what we're testing.